Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -107,13 +107,14 @@ def uninstall_launchctl(*services, command: nil, **_)
         all_services.each do |service|
           ohai "Removing launchctl service #{service}"
           booleans.each do |with_sudo|
+            sudo_user = with_sudo ? "root" : ""
```

Comment:
Passing an empty string in here feels weird.

Reply:
Assuming `with_sudo` is false and `sudo_user` is passed as a separate argument, what should it be set to?

Reply:
Should be `nil` or `"root"` in that case?